### DISCUSSION TITLE
awkward codewars kata solution/comments?

### DISCUSSION THREAD

**Discussion by bhrdj** [codewars/codewars.com#2775] [General]
- I don't want to censor pole dancing as athletics,
    - but the comments below this codewars pole dancing [ascii-art](https://www.codewars.com/kata/reviews/54589d1e36f34f3045000003/groups/5ebec439169a160001d155dc) are kinda lamentable.
- codewars.com can have whatever culture it wants,
    - but i've been recommending many young people, including my 6yo daughter to this site, and I am now reconsidering future recommendations.

**Comment by mitron6** [comment]
It would be unfortunate if you stopped recommending codewars. It is a really great way to learn to code. I had to solve the kata before I could see what you are talking about. Then I googled pole dancing.  My impression is that a person would have to add imagination to make that image a concern and I'd hope younger kids would not do that.... I'm old enough to see why you might worried but young enough not to think it is lamentable. :).

**Reply by mitron6** [reply]
Oh sorry, I guess it wasn't the image but the comments that were concerning.  I think if they are offensive, some of us (with enough honor?) could delete them but I just didn't notice anything a concern. Maybe someone else already deleted them? A little fun is ok but let's not discourage anyone from referring kids to code wars.
